libs/common-utils/src/swap.ts

```diff
@@ -0,0 +1,24 @@
+import { EnvironmentName, environmentName } from './environments'
+
+const DEFAULT_SWAP_URL = 'https://swap.cow.fi'
+
+function _getSwapUrlByEnvironment(): Record<EnvironmentName, string> {
+  return {
+    production: process.env.REACT_APP_DOMAIN_REGEX_PROD || 'https://swap.cow.fi',
```

Comment:
Nitpick:
There is no need to use env variables, its enough to hardcode the values.